src/main/kotlin/org/rust/cargo/toolchain/Cargo.kt

```diff
@@ -97,9 +107,16 @@ class Cargo(
     }
 
     @Throws(ExecutionException::class)
-    fun checkProject(owner: Disposable, projectDirectory: Path): ProcessOutput =
-        CargoCommandLine("check", projectDirectory, listOf("--message-format=json", "--all"))
+    fun checkProject(owner: Disposable, projectDirectory: Path): ProcessOutput {
+        var arguments = listOf("--message-format=json", "--all")
```

Comment:
Let's use `mutableListOf` instead of `listOf `. It allows us to make `arguments` final